méli-mélo/méli-mélo-en.html

```diff
@@ -25,7 +25,7 @@ <h3>Méli-mélo sub-projects</h3>
 <ul>
 {% for feature in melimelo_pages %}
 	{% assign first_example = feature.pages.examples | first %}
-	<li><strong>{{ feature.componentName }}</strong>: <a href="{{ feature.componentName }}/{{ first_example.path }}" lang="{{ first_example.language }}">{{ first_example.title }}</a></li>
+	<li><strong>{{ feature.componentName }}</strong>: <a href="{{ feature.componentName }}/{{ first_example.path }}" lang="{{ first_example.language }}">{{ first_example.title }}</a><!-- {{ feature.url }} --></li>
```

Comment:
What is the goal if this HTML comment? I don't see `url` in any of the metadata files.

Reply:
I added to know why I was seeing twice the same méli-mélo sub projects when I was running the site locally. For now, let's keep it there, it will make easier to identify if this happen for somebody else.